src/code/z_elf_message.c

```diff
@@ -17,7 +17,7 @@ u16 ElfMessage_GetFirstCycleHint(GlobalContext* globalCtx) {
         return 0x224;
     }
     if (!(gSaveContext.weekEventReg[8] & 0x80)) {
-        if (gSaveContext.weekEventReg[9] & 1) {
+        if (gSaveContext.weekEventReg[9] & 0x1) {
```

Comment:
The most common style is to not have hex prefix on single digit numbers. This is down to personal preference and not enforced. I personally don't think these should have hex as a prefix

Reply:
I thought there would be value in making them all consistent one way or another so when we come to replace them all later it's much easier to do.